### ISSUE TITLE
Can't make `SizeToFit: true` work

### ISSUE THREAD

**Issue by bpinto** [colinta/teacup#128]
Can't make sizeToFit method work, what's missing?

Controller:

``` ruby
class HomeController < UIViewController
  stylesheet :home

  layout :root do
    subview(UILabel, :text)
  end
end
```

Style:

``` ruby
Teacup::Stylesheet.new :home do
  style :root,
    backgroundColor: '#2795d9'

  style :text,
    text: 'Test',
    textColor: :white,
    center_x: "50%", top: 100,
    sizeToFit: true
end
```

Result:
![image](https://f.cloud.github.com/assets/526122/1348461/e39e94d8-36c7-11e3-8f2d-c7a63c6e5cec.png)

**Comment by colinta** [comment]
Hmm, someone else reported an issue using it, but I haven't been able to reproduce it.

A (failing) spec would go a long way to helping me or someone to find time to fix this one...

**Comment by bpinto** [comment]
I'll try!

**Comment by MSchmidt** [comment]
Seeing the same. Any progress on this one?

**Comment by colinta** [comment]
It might make a difference if 'sizeToFit' is placed above the position code.

**Comment by colinta** [comment]
(and again, a project that reproduces this error would be a HUGE help)

**Comment by MSchmidt** [comment]
It happens only on the first screen loaded. I'm zipping up my current test project in a sec!

**Comment by MSchmidt** [comment]
Here is a test project: https://github.com/MSchmidt/teacup-label-test
There are 3 labels. Only 2 are visible.
1. positioned with sizeToFit on a subview: visible
2. positioned with sizeToFit on root view: NOT visible
3. positioned with frame on root view: visible

**Comment by colinta** [comment]
Thanks! I don't know why _some_ controllers worked and others didn't, but I was able to fix the problem anyway. Fixed in version 2.1.15, available now. Can you verify?

**Comment by MSchmidt** [comment]
Brilliant! Works like a charm. Thanks for the quick fix!

**Comment by colinta** [comment]
Np!  Actually I found a regression in `center_x` and `center_y` - fixed in 2.1.16.
